### PR TITLE
Allow to config the open, middle, close time for TimeLine

### PR DESCRIPTION
**Pull request by tongchunyin**
And also the number of data point of TimeLine

### CODE REVIEW

File: HSStockChart/TimeLineChart/HSTimeLine.swift [hunk 1]
```diff
@@ -107,6 +122,25 @@ public class HSTimeLine: UIView, HSDrawLayerProtocol {
...
+        self.fiveDayCount = counts
+    }
+    
+    public func setDailyOpenTime(openTime: String, middleTime: String, closeTime: String)
+    {
+        self.openTime = openTime
+        self.middleTime = middleTime
+        self.closeTime = closeTime
+    }
+
```
Comment (zyphs21) [inline HSStockChart/TimeLineChart/HSTimeLine.swift]: You can change the `fileprivate`  for openTime, middleTime and closeTime.

File: HSStockChart/TimeLineChart/HSTimeLine.swift [hunk 2]
```diff
@@ -107,6 +122,25 @@ public class HSTimeLine: UIView, HSDrawLayerProtocol {
...
+    //Getter Setter for customization
+    public func setCountOfTimes(times: Int)
+    {
+        self.countOfTimes = times
+    }
+    
+    public func setfiveDayCount(counts: Int)
+    {
+        self.fiveDayCount = counts
+    }
```
Comment (zyphs21) [inline HSStockChart/TimeLineChart/HSTimeLine.swift]: Change `countOfTimes` and `fiveDayCount` to `public var` is OK. There is no need to make setter function.

File: HSStockChart.podspec [hunk 3]
```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { 'zyphs21' => 'hansenhs21@live.com' }
   s.source           = { :git => 'https://github.com/zyphs21/HSStockChart.git', :tag => s.version.to_s }
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '9.0'
```
[the remark this answers is not in the document]
Comment (tongchunyin) [reply HSStockChart.podspec:9]: That's true

File: HSStockChart/TimeLineChart/HSTimeLine.swift (hunk 2 above)
Comment (tongchunyin) [reply HSStockChart/TimeLineChart/HSTimeLine.swift:135]:
```suggestion
  
```

File: HSStockChart/TimeLineChart/HSTimeLine.swift (hunk 1 above)
Comment (tongchunyin) [reply HSStockChart/TimeLineChart/HSTimeLine.swift:143]:
```suggestion
```

**Pull request merged**